src/main/java/webserver/controller/ReadResourceController.java

```diff
@@ -0,0 +1,57 @@
+package webserver.controller;
+
+import exception.FileNotReadableException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import utils.FileIoUtils;
+import webserver.http.body.DefaultHttpBody;
+import webserver.http.body.HttpBody;
+import webserver.http.header.HttpHeader;
+import webserver.http.message.HttpMessage;
+import webserver.http.message.HttpRequestMessage;
+import webserver.http.request.HttpUri;
+import webserver.http.response.HttpStatus;
+import webserver.http.response.StatusLine;
+
+public class ReadResourceController implements Controller {
+    private static final Logger logger = LoggerFactory.getLogger(ReadResourceController.class);
+
+    @Override
+    public HttpMessage createHttpResponseMessage(HttpRequestMessage httpRequestMessage) {
+        HttpUri httpUri = httpRequestMessage.getRequestLine().getHttpUri();
```

Comment:
```suggestion
        HttpUri httpUri = httpRequestMessage.getHttpUri();
```
이렇게 바꾸면 httpRequestMessage객체에 requestLine가 없을 때 어떻게 처리할지도 결정할 수 있을거에요

Reply:
HTTP Request 명세에 따라 HttpRequestMessage를 만들다보니 Request Line이 존재하지 않거나 명세에 맞지 않으면 
RequestLine 객체를 만들 때 예외를 던지게끔 해두었는데요,
명세상 URI도 Request Line의 구성 요소 중 하나라서 RequestLine이 제대로 만들어지지 않으면 HttpUri를 만들 수 없다고 생각했습니다~

그래서 남겨주신 '_httpRequestMessage객체에 requestLine가 없을 때 어떻게 처리할지_'가 어떤 말씀인지 파악을 못하고 있습니다😢
다시 여쭤봐도 될지요..?

Reply:
httpRequestMessage 객체에서 RequestLine 객체가 없는 경우는 예외 처리해두셨군요
객체에서 객체를 꺼낼 때 생길 수 있는 문제를 예방하자는 이야기 하고 싶었어요 ㅎㅎ;

현재 객체 구조로 보니 저 문제는 없긴 하겠네요.
그러면 읽기 좋은 코드를 위해  객체지향 생활체조의 규칙 4: 한 줄에 점을 하나만 찍는다.를 적용해보면 좋겠네요